modules/essentials/variables.tf

```diff
@@ -44,6 +44,21 @@ variable "worker_iam_role_name" {
   type        = string
 }
 
+########
+# ADOT
+########
+variable "adot_addon_version" {
+  description = "value of the adot addon version"
+  type        = string
+  default     = null
+}
+
+variable "k8s_version_to_set_for_adot" {
```

Comment:
since essentials has required variable `cluster_name`, shall we get the actual cluster version from data resource.. rather than introducing a variable.. ?